tests/furusiya/model/skills/test_whirlwind.py

```diff
@@ -0,0 +1,58 @@
+from unittest.mock import Mock
+
+import math
+import pytest
+
+from model.skills.whirlwind import Whirlwind
+from model.systems.fighter_system import FighterSystem
+
+
+def _mock_factory(*args, **kwargs):
+    m = Mock(*args, **kwargs)
+    m.distance.side_effect = lambda x, y: math.sqrt((x - m.x) ** 2 + (y - m.y) ** 2)
+    return m
+
+
+@pytest.fixture
+def player():
+    yield _mock_factory(x=1, y=2)
+
+
+@pytest.fixture
+def player_fighter():
+    yield Mock()
+
+
+@pytest.fixture
+def bushslime():
+    yield _mock_factory(x=7, y=8)
+
+
+@pytest.fixture
+def _fighter():
+    yield Mock()
+
+
+@pytest.fixture
+def tigerslash():
+    yield _mock_factory(x=2, y=2)
+
+
+@pytest.fixture
+def whirlwind(player, player_fighter, bushslime, tigerslash, _fighter):
+    FighterSystem.set_fighter(player, player_fighter)
+    FighterSystem.set_fighter(bushslime, _fighter)
+    FighterSystem.set_fighter(tigerslash, _fighter)
```

Comment:
Please add a second enemy in range, eg. at `(2, 1)` and assert that both get hit.